crates/swc_css_minifier/src/compressor/rules.rs

```diff
@@ -155,6 +155,21 @@ impl Compressor {
         }
     }
 
+    fn merge_relative_selector_list(
+        &self,
+        left: &RelativeSelectorList,
```

Comment:
Can we take ownership here?
We are unconditionally cloning.

Reply:
@kdy1 I think no because we have `captured outer variable`, but I have ideas how we can rewrite it, can we postpone it for future PRs

Reply:
Yes, I think so